TimeFormat.java

```diff
@@ -13,5 +13,30 @@ public static void main(String[] args) {
 		// Does the same with the minutes part of the input.
 		int minutes = Integer.parseInt("" + args[0].charAt(3) + args[0].charAt(4));
         // Replace this comment with the rest of your code
+		String hours_new = Integer.toString(hours);;

```

Comment:
1. String + int = String
2. use camelCase format for variable names
`String hoursString = hours + "";`
3. Also Don't use Interger.toString(). Wasn't in material.